js/bybit.js

```diff
@@ -5321,6 +5324,31 @@ module.exports = class bybit extends Exchange {
                     'X-BAPI-TIMESTAMP': timestamp,
                     'X-BAPI-SIGN': signature,
                 };
+            } else if (isV3UnifiedMargin) {
```

Comment:
@ttodua Can you check the python version? I'm getting this crash 
![image](https://user-images.githubusercontent.com/43336371/193774012-47d60ebb-57d8-43fb-97f4-63d65258a57a.png)